.circleci/config.yml

```diff
@@ -190,6 +194,7 @@ jobs:
       CLOUD_PROVIDER: "<< parameters.cloud_provider >>"
       KUBECONFIG: /home/circleci/project/data/kubeconfig
       GO111MODULE: "on"
+      NSM_NAMESPACE: "<< parameters.namespace >>"
```

Comment:
This is a global env for the whole job, all the steps get this. No need to set explicitly inside.